CRM/Aip/BAO/AipProcess.php

```diff
@@ -11,5 +11,29 @@
  */
 class CRM_Aip_BAO_AipProcess extends CRM_Aip_DAO_AipProcess
 {
-  
+    /**
+     * @inheritDoc
+     */
+    public static function getSubscribedEvents(): array {
+        return [
+            'civi.afform_admin.metadata' => 'afformAdminMetadata',
+        ];
+    }
+
+    /**
+     * Provides Afform metadata about this entity.
+     *
+     * @see \Civi\AfformAdmin\AfformAdminMeta::getMetadata().
+     */
+    public static function afformAdminMetadata(GenericHookEvent $event): void {
+        $entity = pathinfo(__FILE__, PATHINFO_FILENAME);
+        #$entity = "AipProcess";
```

Comment:
If you want to indicate what value `$entity` has, I'd suggest to remove line 29, and uncomment line 30 (and to use single quotes instead of double quotes). Otherwise remove this line.

Reply:
changed this